app/models/users/notification_setting.rb

```diff
@@ -4,14 +4,13 @@ module Users
   #        destroy callbacks will be called on this object.
   class NotificationSetting < ApplicationRecord
     self.table_name_prefix = "users_"
-    self.ignored_columns = %w[email_connect_messages]
+    self.ignored_columns += %w[email_connect_messages]
 
     belongs_to :user, touch: true
 
     validates :email_digest_periodic, inclusion: { in: [true, false] }
 
     alias_attribute :subscribed_to_welcome_notifications?, :welcome_notifications
-    alias_attribute :subscribed_to_mod_roundrobin_notifications?, :mod_roundrobin_notifications
```

Comment:
This was only being used in an expectation of one spec. Whilst I prefer the naming of this aliased method, I decided to remove it because this method was not being used elsewhere.